.github/workflows/bot_fix_style.yaml

```diff
@@ -0,0 +1,162 @@
+name: bot fix style
+
+# triggers the action when
+on:
+  issue_comment:
+    # the PR receives a comment, or a comment is edited
+    types: [created, edited]
+  pull_request_review:
+    # triggers on a review, whether or not it is accompanied by a comment
+    types: [submitted]
+  pull_request_review_comment:
+    # triggers on a review comment
+    types: [created, edited]
+
+jobs:
+  fix_style:
+    # we set some variables. The ones of the form `${{ X }}${{ Y }}` are typically not
+    # both set simultaneously: depending on the event that triggers the PR, usually only one is set
+    env:
+      AUTHOR: ${{ github.event.comment.user.login }}${{ github.event.review.user.login }}
+      COMMENT_EVENT: ${{ github.event.comment.body }}
+      COMMENT_REVIEW: ${{ github.event.review.body }}
+    name: Fix style issues from lint
+    if: github.event.issue.pull_request
+    # && (startsWith(github.event.comment.body, 'bot fix style') || contains(toJSON(github.event.comment.body), '\nbot fix style'))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find bot fix style
+        id: bot_fix_style
+        run: |
+          COMMENT="${COMMENT_EVENT}${COMMENT_REVIEW}"
+          # we strip `\r` since line endings from GitHub contain this character
+          COMMENT="${COMMENT//$'\r'/}"
+          # for debugging, we print some information
+          printf '%s' "${COMMENT}" | hexdump -cC
+          printf 'Comment:"%s"\n' "${COMMENT}"
+          bot_fix_style="$(printf '%s' "${COMMENT}" |
+            sed -n 's=^bot fix style$=bot-fix-style=p' | head -1)"
+
+          printf $'"bot fix style"? \'%s\'\n' "${bot_fix_style}"
```

Comment:
What exactly does this do?
Do I understand correctly that it checks whether the comment contains a line that *exactly* matches "bot fix style", with nothing before or after it?
If so, that seems good. Because we don't want to run the rest of the workflow if a comment only contains "bot fix style" somewhere in the middle of a line. After all, such a comment is probably a reviewer explaining the functionality to a contributor.